File: nan21.dnet.module.sd/nan21.dnet.module.sd.presenter/src/main/java/net/nan21/dnet/module/sd/opportunity/ds/model/OpportunityPriorityLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.sd.opportunity.ds.model;

import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;

import net.nan21.dnet.module.sd.opportunity.domain.entity.OpportunityPriority;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = OpportunityPriority.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = OpportunityPriorityLovDs.fNAME) })
public class OpportunityPriorityLovDs extends
        AbstractTypeLov<OpportunityPriority> {

    public OpportunityPriorityLovDs() {
        super();
    }

    public OpportunityPriorityLovDs(OpportunityPriority e) {
        super(e);
    }

}

File: nan21.dnet.module.sd/nan21.dnet.module.sd.ui.extjs/src/main/webapp/src/ds/OpportunityPriorityLovDs.js
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */   	 
Ext.define("net.nan21.dnet.module.sd.opportunity.ds.model.OpportunityPriorityLovDs" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"clientId", type:"int", useNull:true},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean"}
	]
});

Ext.define("net.nan21.dnet.module.sd.opportunity.ds.model.OpportunityPriorityLovDsFilter" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean", useNull:true}
	] 
});
	
Ext.define("net.nan21.dnet.module.sd.opportunity.ds.param.OpportunityPriorityLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 